ui-tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "glue-lab-ui-tests",
+  "version": "1.0.0",
+  "description": "JupyterLab glue-lab Integration Tests",
```

Comment:
You forgot to change the name and description.

Reply:
Sorry I missed it.
I opened https://github.com/jupyterlab/jupyter-collaboration/pull/176